db/migrations/20190311150841_add_prisoners_table.js

```diff
@@ -0,0 +1,25 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('prisoners', tbl => {
+        tbl.increments();
+
+        tbl
+            .string('name')
+            .notNullable();
+
+        tbl
+            .integer('id_number')
```

Comment:
We don't need an actual id number for the prisoners. I just meant like a "primary key" type of ID, what you have on line 4 with `tbl.increments()` basically